vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -4,183 +4,184 @@ sidebarDepth: 2
+      * Click `Next`
+5. For `Subnets` → VPC or Virtual Private Cloud subnets should already be setup and selected. Click `Next`.
+6. Click `Next` again
+7. Review the Compute and Scaling configuration. Typically, the AWS defaults below work:
+     * AMI type: `Amazon Linux 2`
+     * Instance type: `t3.medium`
+Click `Next`
+8. Review the Node Group scaling configuration: 
+      * Set `Maximum size` to `5`. This is over-resourced for a Getting Started instance but offers capacity for adding microservices to your cluster without modifying the Nodegroup. 
+      * Click `Next`
+9. Select `Allow remote access to nodes`. Follow the links to create a new SSH key pair if you don't already have one.
+10. Review your settings and then click `Create`. It may take a few minutes for the node to be created.
+ 
 ### Install the NGINX Ingress Controller
-1. Add the NGINX controller for ingress. This depends on your role having permissions for ELB.
-   - For basic NGINX ingress install run this command
-    ```sh
-    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.2/deploy/static/provider/cloud/deploy.yaml
-    ```
-   - See <https://kubernetes.github.io/ingress-nginx/deploy/#aws> as well as [this](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more detailed install steps.
-2. Get the ELB external URL for your NGINX install
-   - Run: ```kubectl get services -n ingress-nginx```
-   - Get the value of the external address (EXTERNAL-IP) for the ingress-nginx-controller:
-```
-NAME                                 TYPE           CLUSTER-IP       EXTERNAL-IP                        
-ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1ff4dadb44639a6bbf707e-0e0a483d966405ee.elb.us-east-2.amazonaws.com
-```
+Add the NGINX controller for the ingress. This depends on your role having permissions for ELB.
 
+1. From the EKS cloud shell (or AWS client), login.
+2. Apply this version of NGINX:
+```sh
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.5/deploy/static/provider/cloud/deploy.yaml 
+```
 ::: tip
 NGINX is working correctly if a `404 Not Found` error page is generated when accessing the EXTERNAL-IP from your browser. Alternatively, you can [set up a simple test application](../devops/manage-nginx.md#verify-the-nginx-ingress-install) using your local `kubectl`. You can also [customize the NGINX ingress](../devops/manage-nginx.md#customize-the-nginx-configuration) to optimize the configuration for Entando.
 :::
+
+See the K8s [Load Balancer Install Guide for AWS](https://kubernetes.github.io/ingress-nginx/deploy/#aws) and the [NGINX as Ingress Controller on EKS](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more details.
```

Comment:
"the NGINX as Ingress Controller on EKS" is awkward without "guide" or "tutorial"